### PR TITLE
(fix) O3-3480 removed optional translation modules in some config options

### PR DESCRIPTION
**Pull request by suubi-joshua**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-docs.openmrs.org/docs/frontend-modules/contributing.en-US#contributing-guidelines) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://om.rs/o3ui).
- [ ] My work includes tests or is validated by existing tests.

## Summary
I removed the the optional translation modules that provide custom i18n In the service queues app and ward app in the config schema,

## Related Issue

https://openmrs.atlassian.net/browse/O3-3480

### CODE REVIEW

**Review by chibongho — CHANGES_REQUESTED** [review]
This change is not sufficient. We also need to modify places where `headerI18nModule` (and similar variables) are referenced.

**Comment by suubi-joshua** [comment]
> This change is not sufficient. We also need to modify places where `headerI18nModule` (and similar variables) are referenced.

Ohhh yes my bad let me work on that @chibongho

File: packages/esm-service-queues-app/src/config-schema.ts [hunk 1]
```diff
@@ -258,11 +258,6 @@ export const configSchema = {
             'The header text for the column. Will be translated if it is a valid translation key. If not provided, the header will be based on the columnType.',
           _default: null,
         },
-        headerI18nModule: {
```
Comment (chibongho) [inline packages/esm-service-queues-app/src/config-schema.ts:261]: still need to make corresponding change where `headerI18nModule` is used.

Comment (suubi-joshua) [reply packages/esm-service-queues-app/src/config-schema.ts:261]: Fixed this in the latest commit

**Comment by brandones** [comment]
@suubi-joshua looks like there are some merge conflicts. Can you get this in good shape and then either @chibongho or I will provide a review?

Sorry it got stuck for three weeks. If you have an existing reviewer for a PR and don't hear back for a week or two, you should feel free to ping them.

**Comment by suubi-joshua** [comment]
> @suubi-joshua looks like there are some merge conflicts. Can you get this in good shape and then either @chibongho or I will provide a review?
> 
> Sorry it got stuck for three weeks. If you have an existing reviewer for a PR and don't hear back for a week or two, you should feel free to ping them.

Okay let me resolve conflicts. Will surely do in the future thanks @brandones

**Comment by suubi-joshua** [comment]
@brandones made some changes and fixed the conflicts.

**Review by chibongho — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**